lib/chat2/zendesk_chat2.dart

```diff
@@ -51,7 +51,7 @@ class Zendesk2Chat {
     Map arguments = {
       'accountKey': accountKey,
       'appId': appId,
-      if (iosThemeColor != null) 'iosThemeColor': iosThemeColor.value,
```

Comment:
you should not remove the conditional `iosThemeColor`. Even if it is obsolete, it may crash if the value is `null`, please revert this

Reply:
Hello,
With the default value you are setting and null safety it can no longer be null.
The lint marked it as a unnecessary check for null

If you declared it as Color? it would be necessary
 ``` dart
Future<void> init(
    String accountKey,
    String appId, {
    @Deprecated('Prefer to use custom UI chat providers')
        Color? iosThemeColor = Colors.indigo,
  })
```

 do you still want me to revert the changes?